### PR TITLE
Posted answers for Git questions

### PR DESCRIPTION
**Pull request by shanmugapriyan98**
Answered the four questions on basics of git.

### CODE REVIEW

File: Answers.txt [hunk 1]
```diff
@@ -1,7 +1,29 @@
 • What is a gitignore file and what should it include for a Java project? 
 
-• What does fast-forward in git mean? 
+Gitignore file is used for ignoring certain types of files during commit.
```
Comment (pskeerth) [inline Answers.txt:3]: Do the gitignore files get pushed when we do a git push?

Comment (shanmugapriyan98) [reply Answers.txt:3]: No, the main reason for using gitignore file is to ignore these files during commit and push

Comment (pskeerth) [reply Answers.txt:3]: okay

**Review by pskeerth — APPROVED** [review]
Looks good!

**Pull request merged**